include/translated-object.php

```diff
@@ -456,4 +456,82 @@ protected function validate_translations( $translations, $id = 0, $context = 'sa
 		/** @phpstan-var array<non-empty-string, positive-int> */
 		return array_merge( array( $lang->slug => $id ), $translations );
 	}
+	/**
+	 * Creates translations groups in mass.
+	 *
+	 * @since 1.6.3
+	 * @since 3.4 Moved from PLL_Admin_Model class.
+	 *
+	 * @param array  $translations Array of translations arrays.
```

Comment:
Can we better precise the type? Also remove extra spaces.

Reply:
Done in a56a670593dda2e4ac16f94fcdf079b4086ce5bc